README.md

```diff
@@ -92,6 +93,8 @@ struct ContentView_Previews: PreviewProvider {
 }
 ```
 
+This app has a `Start Stream` button which starts monitoring a demo stream in a streaming screen. This streaming screen provided by this kit is self contained. It has the necessary  components for monitoring a streaming. It has a settings menu, in which it provides different video views, audio selections, and streaming sorting orders.
```

Comment:
Double blank space in this bit

`It has the necessary  components`

Reply:
> This app has a `Start Stream` button which starts monitoring a demo stream in a streaming screen. This streaming screen provided by this kit is self contained. It has the necessary  components for monitoring a streaming. It has a settings menu, in which it provides different video views, audio selections, and streaming sorting orders.

to 

This app has a `Start stream` button which starts playing a demo stream in the app. The streaming screen contains all the components you need in order to build your own experience - including a settings menu where you can select different video layouts, audio source selection and stream sort order.